garak/detectors/dropboxrepeat.py

```diff
@@ -0,0 +1,47 @@
+from garak.detectors.base import Detector
+import re
+
+class DropboxRepeatedTokenDetector(Detector):
+    """
+    Detects hallucinations and divergence triggered by repeated token inputs.
```

Comment:
please describe in more detail how the probe works, in a second line after this summary

Reply:
done.